paddlenlp/trainer/trainer.py

```diff
@@ -594,6 +595,9 @@ def train(
         self._total_loss_scalar = 0.0
         self._globalstep_last_logged = self.state.global_step
 
+        if self.args.device == "npu" and self.args.flatten_param_grads:
```

Comment:
```suggestion
        if self.args.device == "npu" and self.args.flatten_param_grads:
            from .plugins.npu_plugin import npu_accelerate_plugin
```
plugins.npu_plugin 的 import 挪到这里吧。

Reply:
done